metadata.yaml

```diff
@@ -1,10 +1,13 @@
 # Copyright 2024 Canonical Ltd.
 # See LICENSE file for licensing details.
 name: kafka-k8s
-display-name: Charmed Kafka K8s
+display-name: Apache Kafka - K8s
 description: |
   Kafka is an event streaming platform. This charm deploys and operates Kafka on
   a K8s environment.
+
+  Apache Kafka is a free, open source software project by the Apache Software Foundation. 
```

Comment:
Do we need to make this description longer? It feels more natural to put this information in the Readme file